### PR TITLE
Biocollect 98

### PR DESCRIPTION
**Pull request by sat01a**

### CODE REVIEW

File: grails-app/services/au/org/ala/biocollect/merit/ActivityService.groovy [hunk 1]
```diff
@@ -78,6 +78,11 @@ class ActivityService {
         webService.doDelete(grailsApplication.config.ecodata.service.url + '/activity/' + id)
     }
 
+    def isUserOwnerForActivity(userId, activityId) {
+        def params = '&activityId=' + activityId
+        webService.doDelete(grailsApplication.config.ecodata.service.url + '/activity/isUserOwnerForActivity/' + userId + params)
```
Comment (m-r-c) [inline grails-app/services/au/org/ala/biocollect/merit/ActivityService.groovy]: This should be a get, not a delete.

File: grails-app/controllers/au/org/ala/biocollect/RecordController.groovy [hunk 2]
```diff
@@ -21,6 +30,8 @@ class RecordController {
         }
         model.records = results?.list
         model.total = results?.total
+        String userId = userService.getCurrentUserId()
+        model.showCrud = userId && id ? projectService.isUserAdminForProject(userId, id) : false
```
Comment (m-r-c) [inline grails-app/controllers/au/org/ala/biocollect/RecordController.groovy]: Does this mean we only show the add/edit/delete options for project admins? Should also be for the owner of the activity/record, shouldn't it?

File: grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy [hunk 3]
```diff
@@ -223,6 +258,8 @@ class BioActivityController {
         }
         model.activities = results?.activities
         model.total = results?.total
+        String userId = userService.getCurrentUserId()
+        model.showCrud = userId ? projectService.isUserAdminForProject(userId, id) : false
```
Comment (m-r-c) [inline grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy]: Does this mean we only show the add/edit/delete options for project admins? Should also be for the owner of the activity/record, shouldn't it?

File: web-app/js/records.js [hunk 4]
```diff
@@ -36,14 +38,42 @@ var RecordListsViewModel = function(){
     });
   };
 
+  self.delete = function (record) {
+    bootbox.confirm("Are you sure you want to delete the records?", function (result) {
```
Comment (m-r-c) [inline web-app/js/records.js]: Shouldn't the message be "Are you sure you want to delete this record?" -> You're only deleting 1.

Comment (sat01a) [inline web-app/js/records.js]: Fixed

File: grails-app/services/au/org/ala/biocollect/merit/ActivityService.groovy (hunk 1 above)
Comment (sat01a) [inline grails-app/services/au/org/ala/biocollect/merit/ActivityService.groovy]: fixed

File: grails-app/controllers/au/org/ala/biocollect/RecordController.groovy (hunk 2 above)
Comment (sat01a) [inline grails-app/controllers/au/org/ala/biocollect/RecordController.groovy]: Ya, I will check with Peter to see whether we can have that.

**Pull request merged**